internal/cli/auth/register.go

```diff
@@ -61,9 +61,9 @@ Run '%s auth register --profile <profileName>' to use your username and password
 		Args: require.NoArgs,
 	}
 
-	cmd.Flags().BoolVar(&opts.isGov, "gov", false, "Register to Atlas for Government.")
-	cmd.Flags().BoolVar(&opts.noBrowser, "noBrowser", false, "Don't try to open a browser session.")
-	cmd.Flags().BoolVar(&opts.skipConfig, "skipConfig", false, "Skip profile configuration.")
+	cmd.Flags().BoolVar(&opts.IsGov, "gov", false, "Register to Atlas for Government.")
+	cmd.Flags().BoolVar(&opts.NoBrowser, "NoBrowser", false, "Don't try to open a browser session.")
```

Comment:
seems like your find and replace over did it

Reply:
yep, and I didn't caught this one cause we don't generate docs, thank you!